integration_tests/steps/integration_tests.py

```diff
@@ -1059,6 +1059,8 @@ def identical_cumulative_bills(context):
 def identical_profiles(context):
     device_stats_dict = context.simulation.endpoint_buffer.device_statistics.device_stats_dict
     load_profile = device_stats_dict['House 1']['H1 DefinedLoad']['load_profile_kWh']
-    for time_slot, value in load_profile.items():
-        if time_slot.add(days=1) < today(tz=TIME_ZONE).add(days=2):
-            assert value == load_profile[time_slot.add(days=1)]
+    load_profile_ts = {int(from_format(time_slot, DATE_TIME_FORMAT).timestamp()): value
+                       for time_slot, value in load_profile.items()}
+    start = int(list(load_profile_ts.keys())[0])
+    end = int(list(load_profile_ts.keys())[23])
+    assert all([load_profile_ts[i] == load_profile_ts[i + 86400] for i in range(start, end, 3600)])
```

Comment:
This ps probably not needed any more, right?
It may be my opinion, but the old implementation is more human readable.

Reply:
In old implementation `time_slot ` was in and now it is in string, so now how had to parse it in any format anyway for integration test, so timestamp seems better to me with less hustle

Reply:
Ok, this makes sense. Thanks!